pup.js

```diff
@@ -0,0 +1,24 @@
+const puppeteer = require('puppeteer');
+const fs = require('fs');
+
+const url = 'http://localhost:3000'
+
+if (!url) {
+  throw "Please provide URL as a first argument";
+}
+async function run () {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto(url, {waitUntil: 'networkidle0'});
+  await page.screenshot({path: 'screenshot.png'});
```

Comment:
what is this screenshot used for?

Reply:
Good question! I think it was used merely to test this beast. Not required anymore.